environment.yml

```diff
@@ -3,7 +3,7 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - python=3.8
+- python=3.8
```

Comment:
didn't we fix this already?

Reply:
yes, seems to be the same problem with the not correct merge.